observability/test/manual_deploy.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
One thing I don't see in this PR is documentation. It will take a lot of pressure off of you to be the only person who can operate the interop framework if you document:

- what the most important files _do_
- how to run them locally
- if you _can't_ easily run locally, how to set up an env in which you _can_ run them

Take a look at [this](https://github.com/grpc/grpc/blob/master/tools/run_tests/xds_k8s_test_driver/README.md) for an example

Reply:
Yes I plan to add those instructions in the next set of PRs. One of the reasons I haven't done it so far is that _the way things work_ have been changing a lot. So I had wanted things to stabilize a bit before adding those instructions thinking they are going to get completely overhauled anyways.